### PR TITLE
upload media from service

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
@itkg-ngilain @itkg-alavieille

File: MediaBundle/Resources/config/manager.yml [hunk 1]
```diff
@@ -21,3 +22,9 @@ services:
         arguments:
             - @knp_gaufrette.filesystem_map
             - %open_orchestra_media.filesystem%
+    open_orchestra_media.manager.save_media:
+        class: %open_orchestra_media.manager.save_media.class%
+        arguments:
+            - %open_orchestra_media.tmp_dir%
+            - @open_orchestra_media.thumbnail_manager
+            - @open_orchestra_media.manager.uploaded_media
```
Comment (itkg-alavieille) [inline MediaBundle/Resources/config/manager.yml]: eof

**Comment by itkg-rboessel** [comment]
@itkg-alavieille  done

File: Media/EventListener/MoveUploadedFileListener.php [hunk 2]
```diff
@@ -3,62 +3,20 @@
...
-        $this->thumbnailManager = $thumbnailManager;
-        $this->uploadedMediaManager = $uploadedMediaManager;
-    }
+    protected $saveFileManager;
 
-    /**
-     * @param LifecycleEventArgs $event
-     */
-    protected function preUpload(LifecycleEventArgs $event)
+    public function __construct(SaveMediaManagerInterface $saveFileManager)
```
Comment (itkg-alavieille) [inline Media/EventListener/MoveUploadedFileListener.php]: comment

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**